api/lib/source/polkadotV0-source.js

```diff
@@ -667,6 +678,224 @@ class polkadotAPI {
     }
   }
 
+  getDemocracyProposalDetailedVotes(proposal, links) {
+    // in democracy proposals there is the first opening deposit made by the proposer
+    // afterwards every account that seconds the proposal must deposit the same amount from the initial deposit
+    const depositsSum = toViewDenom(
+      this.network,
+      BigNumber(proposal.balance).times(proposal.seconds.length).toNumber()
+    )
+    const deposits = [
+      {
+        depositer: proposal.proposer,
+        amount: [
+          {
+            amount: toViewDenom(this.network, proposal.balance),
+            denom: this.network.stakingDenom
+          }
+        ]
+      }
+    ].concat(
+      proposal.seconds.map((second) => ({
+        depositer: second,
+        amount: [
+          {
+            amount: toViewDenom(this.network, proposal.balance),
+            denom: this.network.stakingDenom
+          }
+        ]
+      }))
+    )
+    const votes = proposal.seconds.map((secondAddress) => ({
+      voter: secondAddress,
+      option: `Yes`
+    }))
+    const votesSum = proposal.seconds.length
+    return {
+      deposits,
+      depositsSum,
+      votes,
+      votesSum,
+      votingPercentageYes: `100`,
+      votingPercentagedNo: `0`,
+      links,
+      timeline: [{ title: `Proposal created`, time: proposal.creationTime }]
```

Comment:
only one object in the timeline?

Reply:
welcome to Polkadot :shrug: better one than nothing at all, I thought